src/main/java/mate/academy/hibernate/relations/exceptions/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.hibernate.relations.exceptions;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
+        super(message);
+    }
+}
```

Comment:
Consider adding an additional constructor to `DataProcessingException` that accepts both a `String` message and a `Throwable` cause. This will allow you to pass the original exception when throwing a `DataProcessingException`, providing more context for debugging.